### ISSUE TITLE
RP-0: Post modern-day

### ISSUE THREAD

**Issue by pjf** [KSP-RO/RP-0#99]
This is something I keep harping on about, so I'm opening an issue as a place where discussion can happen.

I would like to see RP-0 provide gameplay not just up to the modern day, but also beyond it. One of my favourite things about RPL is that it integrated with KSP Interstellar, extraplanetary launchpads, and other "future" technology.

Playing with future tech isn't just something I find deeply satisfying, but it also encourages players to think about what the future *might* be like if Earth had a strong and vibrant space programme, including off-world colonisation and industry. I view this as an essential stepping-stone for humanity's future.

With this in mind, I want to make sure that we plan RP-0 to ensure that future tech is supported and balanced. I'm expecting a lot of the right-hand-side of the tech-tree to be modern and future technologies; hitting the right-hand-edge should be hitting some pretty awesome future tech, and modern day technology should be somewhere in the middle of the tech tree. In many cases, I'd liketo see technology unlocks at times when they *were* possible, if funding were available to carry out those missions. The unlocks themselves can be quite expensive in terms of research and science, representing that some projects have lofty goals.

**TL;DR:** Stock KSP does a great job of inspiring people and fuelling their imaginations of what the future may be like. I'd like to ensure RP-0 can do the same.

**Comment by jwvanderbeck** [comment]
While I don't disagree I really feel like this, and some other stuff
lately, is really getting ahead of yourself.  I don't want to see RP-0 turn
into the nightmare that RO is, or was before serious time was spent
cleaning it up.

Personally my approach to contributing to RP-0 has been very methodical and
focusing on working TL by TL starting at the start of the game.

I completely agree with you on future tech, I just thing we need a solid
base first before jumping to the right hand side of the tree.

- John Vanderbeck
- http://www.johnvanderbeck.com

**Comment by pjf** [comment]
@jwvanderbeck : I'm honestly confused by your comment. I'm definitely not suggesting you personally do the part placement and costing of future parts, but I *do* want to make sure we don't discover later on that existing placements aren't incompatible with future technology.

As for placing parts on the right side of the tree, this has always been my focus since RP-0 was launched, and one of the reasons why I pushed for us to adopt the Community Tech Tree. Most of my commits have been to the left hand side is because we *do* need that stable base, but needing a stable base isn't incompatible with supporting future technology in career mode.

**Comment by jwvanderbeck** [comment]
Well at the end of the day its YOUR project.  My main concern is the play
experience and right now its strong at the start, then weak.  Why go right
to post modern tech when the middle section still needs to be fleshed out?

- John Vanderbeck
- http://www.johnvanderbeck.com

**Comment by pjf** [comment]
> Well at the end of the day its YOUR project.

I hope not! :) RP-0 was something I helped nudge into existence after many discussions with @NathanKell, but at the end of the day I most sincerely hope that it's the community's project, and what's best for the community takes precedence over any crazy ideas I might have.

> My main concern is the play experience and right now its strong at the start, then weak. Why go right to post modern tech when the middle section still needs to be fleshed out?

Oh! We might be in agreement then. :) The reason I'm thinking about future tech is:

1. I most sincerely hope that the middle section gets fleshed out, and so I want to make sure the late tech tree isn't adversely impacted by that. (Having the right-hand-side be "modern day" rather than "future technology" would be an example of this.) RPL was great in that it extended career mode well into the future.
1. A lot of "future tech" is past tech that didn't happen. Nuclear pulse propulsion was looking to be a very viable technology until bans until the Partial Test Ban Treaty got in the way. Small nuclear reactors have been in use since the 1950s in submarines. Microwave power transmission was demonstrated as viable in the 1960s. There's a *lot* of scope for some quite vibrant technology to be in the middle of the tree.

The ability to place historically viable but undeveloped technology in the middle of the tree is *really* exciting stuff. RP-0 gives us a chance to allow players to explore some amazing histories that never were, as well as some glorious futures that may still await. <3

**Comment by BevoLJ** [comment]
> The ability to place historically viable but undeveloped technology in the middle of the tree is really exciting stuff. RP-0 gives us a chance to allow players to explore some amazing histories that never were, as well as some glorious futures that may still await. <3

I recently added the M1 engines to RP-0 and wow, how I wish those hadn't had their funding cut IRL!  :D 

For me the biggest upside of including things like the M1 is that while there are limitations IRL that we can get around due to this being a game, there are also many limitations on the game due it being simulated in a software physics engine.  For example IRL some stages had a dozen (even dozens) of engines on single stages. In the game trying to build a large multistage rocket with dozens of engines on each stage is from my experience not a good plan. Especially if running recording software, using OpenGL, and extra risky if playing with KCT installed as KJR not working can be very expensive.

Using things like the M1 do offer more options to deal with issues like keeping low engine and part counts that the game needs in order to run properly.

**Comment by pap1723** [comment]
This will always be something we need. More Modern Day and future ideas.

**Comment by Bornholio** [comment]
If i get some real time to spend I'd like to wholesale get Jplrepos near future mod parts in to bluesky
